src/ngraph_builder.cc

```diff
@@ -73,12 +73,11 @@ static tf::Status ValidateInputCountMin(const tf::Node* op, size_t count) {
 //    shared_ptr<ng::Node> output_node - ng::Node to store
 //
 
-static tf::Status SaveNgOp(Builder::OpMap& ng_op_map, const std::string op_name,
-                           const shared_ptr<ng::Node> output_node) {
+static void SaveNgOp(Builder::OpMap& ng_op_map, const std::string& op_name,
+                     const shared_ptr<ng::Node>& output_node) {
```

Comment:
Good catch with `op_name`. In terms of `output_node`, it's in fact a pointer so there shouldn't be much difference
(tensorflow code has both passing by ref and by value).  Ok to change it.